site/content/blog/carvel-in-july-2022.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Carvel In July"
+slug: carvel-in-july-2022
+date: 2022-08-04
+author: Joe Kimmel, Praveen Rewar, and Carvel Engineers
```

Comment:
Okay, so I was just wondering that this might look like a really long line in the website and just "Joe Kimmel and Carvel Engineers" would also suffice and I went ahead and looked at the preview generated by Netlify. I realised that the author section is actually a hyperlink and when we click on it, it looks for blogs that have a tag with the same name as the author name. A few of our initial blogs do have the author name also present in the tags list, but we have recently stopped that and because of this that hyperlink in most of the blogs now leads to a not found page 😞